bot/exts/utils/bot_stats.py

```diff
@@ -32,15 +32,15 @@ async def stats(self, ctx: commands.Context) -> None:
             color=Colours.green,
         )
 
-        embed.set_thumbnail(url=self.bot.user.avatar_url)
+        embed.set_thumbnail(url=self.bot.user.display_avatar.url)
 
         uptime = humanize.precisedelta(
             datetime.utcnow().timestamp() - self.bot.launch_time
         )
 
         fields = {
             "Python Version": python_version(),
```

Comment:
```suggestion
            "Python version": python_version(),
```
I think this looks nicer

Reply:
https://github.com/gurkult/gurkbot/pull/155/commits/7ab5b706e62acc02ae59d3e658b242f9c8c728f9

Reply:
https://github.com/gurkult/gurkbot/pull/155/commits/7ab5b706e62acc02ae59d3e658b242f9c8c728f9